### PR TITLE
Rb starting on UI

### PR DESCRIPTION
**Pull request by BenjaminGuzman-Ayala**

### CODE REVIEW

**Review by jpanto21 — APPROVED** [review]
good 😃👍

File: config/routes.rb [hunk 1]
```diff
@@ -1,10 +1,11 @@
 Rails.application.routes.draw do
+  root "photos#index"
```
Comment (JJohnson606) [inline config/routes.rb:2]: Good job putting your root directory at the top.

File: lib/tasks/dev.rake [hunk 2]
```diff
@@ -10,6 +10,20 @@ task({ :sample_data => :environment }) do
...
+  
+    usernames.each do |username|
+      User.create(
+        email: "#{username}@example.com",
+        password: "password",
+        username: username.downcase,
+        private: [true, false].sample,
+      )
+    end
+
```
Comment (JJohnson606) [inline lib/tasks/dev.rake:26]: A nice and simple backend. looks good.

**Review by JJohnson606 — COMMENTED** [review]
Looks great.

**Pull request merged**